examples/lichess.rs

```diff
@@ -51,7 +51,7 @@ struct App {
     lichess: Lichess,
 }
 
-#[tokio::main]
+#[tokio::main(flavor = "current_thread")]
```

Comment:
```suggestion
#[tokio::main]
```

IIUC, if we have the `rt` runtime, I don't think this is necessary as it can only be single threaded

Reply:
It looks like it's necessary after all. Check the failed [test](https://github.com/ion232/lichess-api/actions/runs/10800358143/job/29958167993#step:6:303).

Reply:
That's odd, I tested this myself locally. Fair enough though